src/block-kit/Modal.tsx

```diff
@@ -0,0 +1,46 @@
+/** @jsx JSXSlack.h */
+import { View } from '@slack/types'
+import { JSXSlack } from '../jsx'
+import { ObjectOutput } from '../utils'
+import { Blocks, BlockComponentProps } from './Blocks'
+import { plainText } from './composition/utils'
+
+// TODO: Use original View type when supported fields for API on @slack/types
+type ViewForAPI = View & {
+  callback_id?: string
+  external_id?: string
+  hash?: string
```

Comment:
`hash` is not a part of view JSON and only for payloads in `views.update`. It will throw an error if specified hash as view JSON.